### ISSUE TITLE
Unable to instantiate activity

### ISSUE THREAD

**Issue by andriybashta** [StephenVinouze/KinApp#6]
Hello. I try to install your library to my project and get this error:
<img width="1440" alt="pf insta-filterspf - app 2019-01-02 21-35-00" src="https://user-images.githubusercontent.com/1914602/50608846-85144880-0ed6-11e9-84bc-a2f97cc09772.png">

Please, help.

**Comment by StephenVinouze** [comment]
Hi, sorry for the late reply
Just made some clean in the library as I didn't have much time to deal with it lately
Turns out that the pr i merged was not working properly
Put stable coroutine version and fix the build
Seeing the code made me realise that this library ties the user by using coroutine. This is really bad architecture and i should take some time in changing how i implemented the coroutine. I did it when it was in early stage so was hard to find some docs but it's not the case anymore.
I published a fix version in the meanwhile (1.0.3) and will publish another when this coroutine binding will be fixed
